css/fonts/icomoon.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
This is a nice reference file to have, but I would not put it in the `css` dir. Perhaps in a section for the styleguide would make more sense.